src/status_im/ui/screens/views.cljs

```diff
@@ -91,8 +95,13 @@
                           :accounts accounts
                           :login login
                           :recover recover
+                          :network-settings network-settings
+                          :paste-json-text paste-json-text
+                          :add-rpc-url add-rpc-url
+                          :network-details network-details
```

Comment:
Value align

Reply:
That doesn't work and has never worked for `case` form. I'm not going to add this manually.